README.md

```diff
@@ -2,7 +2,7 @@
 ![Auth0Mautic](https://www.leuchtfeuer.com/fileadmin/github/auth0-for-mautic/Mautic-Auth0.png "Auth0 for Mautic")
 
 
-[![Latest Stable Version](https://poser.pugx.org/leuchtfeuer/mautic-auth0-bundle/v/stable)](https://packagist.org/packages/leuchtfeuer/mautic-auth0-bundle)
+[![Latest Stable Version](https://poser.pugx.org/leuchtfeuer/mautic-leuchtfeuerauth0-bundle/v/stable)](https://packagist.org/packages/leuchtfeuer/mautic-auth0-bundle)
```

Comment:
@beetofly Does this URL `https://poser.pugx.org/leuchtfeuer/mautic-leuchtfeuerauth0-bundle/v/stable` already exist (or will be)?

Reply:
@Moongazer Resolved with commit ""MTC-3811-update-readme"